addons/l10n_dk/models/template_dk.py

```diff
@@ -32,6 +32,8 @@ def _get_dk_res_company(self):
                 'account_journal_early_pay_discount_gain_account_id': 'dk_coa_2720',
                 'account_sale_tax_id': 'tax_s1',
                 'account_purchase_tax_id': 'tax_k1',
+                'default_cash_difference_income_account_id': 'dk_coa_2720',
```

Comment:
I think this is related to the part that stays in master

Reply:
:man_facepalming: